client/src/components/common/Header.tsx

```diff
@@ -0,0 +1,193 @@
+                        <Menu.Button className='relative flex max-w-xs items-center rounded-full bg-transparent text-sm'>
+                          <p className='mr-3 hidden text-left text-lg sm:block'>
+                            <strong className='block font-medium text-whiteTextHeader'>
+                              {user.name}
+                            </strong>
+                          </p>
+
+                          <div className='relative h-12 w-12'>
+                            <Image
+                              alt='Man'
+                              src={user.imageUrl}
+                              className='rounded-full shadow-sm'
+                              width={100}
+                              height={100}
+                            />
+                          </div>
+
+                          <ArrowDownIcon className='mr-3 ms-4 hidden h-5 w-5 text-white transition group-hover:text-gray-700 sm:block' />
+                        </Menu.Button>
+                      </div>
+                      <Transition
+                        as={Fragment}
+                        enter='transition ease-out duration-100'
+                        enterFrom='transform opacity-0 scale-95'
+                        enterTo='transform opacity-100 scale-100'
+                        leave='transition ease-in duration-75'
+                        leaveFrom='transform opacity-100 scale-100'
+                        leaveTo='transform opacity-0 scale-95'
+                      >
+                        <Menu.Items className='absolute right-0 z-10 mt-2 w-48 origin-top-right rounded-md bg-white py-1 shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none'>
+                          {userNavigation.map((item) => (
+                            <Menu.Item key={item.name}>
+                              <a
+                                href={item.href}
+                                className='block px-4 py-2 text-sm text-gray-700 active:bg-gray-100'
+                              >
+                                {item.name}
+                              </a>
+                            </Menu.Item>
+                          ))}
+                        </Menu.Items>
+                      </Transition>
+                    </Menu>
+                  </div>
+                </div>
+                <div className='-mr-2 flex md:hidden'>
+                  {/* Mobile menu button */}
+                  <Disclosure.Button className='relative m-5 inline-flex items-center justify-center rounded-md bg-transparent p-2 text-gray-400 hover:text-white'>
+                    {open ? (
+                      <CrossIcon className='h-8 w-8' />
+                    ) : (
+                      <MenuIcon className='h-8 w-8' />
+                    )}
+                  </Disclosure.Button>
+                </div>
+              </div>
+            </div>
+
+            {/* opciones del menu en mobile */}
```

Comment:
Dejaría los comentarios en el código en inglés 🙂 que fue medio lo que vinimos haciendo tanto en el BE como en el FE

Reply:
Joya! ahí quedo